recipe/meta.yaml

```
@@ -0,0 +1,66 @@
+{% set name = "mcl" %}
+{% set version = "1.86.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+
+  url: https://github.com/herumi/{{ name}}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: a3ab1cd01a6419f46210fd5874e795e37a4565fa305e0302301b31ffed208872
+
+build:
+  skip: True  # [s390x]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - clang                                                 # [linux and aarch64]
+    - clangxx                                               # [linux and aarch64]
+    - cmake
+    - make                                                  # [unix]
+    - ninja                                                 # [win]
```

```suggestion
    - ninja-base                                                 # [win]
```
`ninja` includes `python` but it seems you don't need it